src/plugins/workspace/server/utils.test.ts

```diff
@@ -5,7 +5,18 @@
 
 import { AuthStatus } from '../../../core/server';
 import { httpServerMock, httpServiceMock } from '../../../core/server/mocks';
-import { generateRandomId, getPrincipalsFromRequest } from './utils';
+import {
+  generateRandomId,
+  getApplicationOSDAdminConfig,
+  getOSDAdminConfig,
+  getPrincipalsFromRequest,
+  stringToArray,
+  updateDashboardAdminStateForRequest,
+} from './utils';
+import { getWorkspaceState } from '../../../core/server/utils';
+import { AppPluginSetupDependencies } from './types';
+import { Observable, of } from 'rxjs';
+import { error } from 'console';
```

Comment:
What's this used for?

Reply:
I will use `throw new Error('')` to replace it.